src/main/java/seedu/address/model/issue/Priority.java

```diff
@@ -8,7 +8,7 @@
 public enum Priority {
     NONE, LOW, MEDIUM, HIGH;
 
-    public static final String MESSAGE_CONSTRAINTS = "PRIORITY NOT IMPLEMENTED";
+    public static final String MESSAGE_CONSTRAINTS = "Priority should be an integer 0, 1 or 2";
```

Comment:
A minor suggestion: might be helpful to mention what it is for : for e.g. 1(MEDIUM)